specs/devexperience/DevExperience.md

```diff
@@ -0,0 +1,181 @@
+3.	There must be an Item Template for the package.appxmanifest
+4.	The default splash screen and start menu assets will be provided by the tooling. The same behavior of preventing store submissions for apps which use these assets will still exist. 
+5.	A migration tool will be provided for migrating projects
+    - Work for this has started with the https://github.com/dotnet/try-convert tool.
+    - This tool needs to be updated for .vcxproj.
+    - This tool will update code, project files, and manifest files
+    - Update nuget references to WinUI3 supported packages (i.e. Win2D & WCT)
+6.	The existing Manifest designer tool in VS must work for these projects
+7.	The existence of a Package.appxmanifest file will enable Packaged Desktop apps by default.
+8.	The Reunion NuGet package will provide tooling for generating the Sparse Package and Dynamic Dependencies registration code. (link to existing issues)
+a.	[.NET Only] A source generator will be provided to accomplish this
+b.	[C++ only] A build task will be provided for this
+9.	The Reunion meta package will provide tooling for generating typed access to Resource strings for resw resources, similar to how resx works in .NET
+a.	[.NET Only] A source generator will be provided this.
+b.	[C++ only] A build task will be provided for this
+10.	The existing Publish UI will be enabled for creating and publishing app packages to the Microsoft Store.
+
+
+## 5.0 - Developer Workflow
+This section goes into more fine grain detail about the developer experience that will come with developers who are using Project Reunion. It will expand off the Key Design Elements and Principles sections, and articulate where each of those comes into play in the overall story of Reunion.
+
+### 5.1 – Migration and Compat
+This section will highlight the basic workflow for developers looking to use Reunion, as well as highlight some basic high-level requirements that the tooling must guarantee. The section on MSIX packaging will go into more detail on how we will simplify the developer story, while maintaining compat at the same time.
+
+#### 5.1.1 - Migrating Existing Win32 Apps
+App developers which are migrating existing Win32 Apps will do the following:
+1.	Install Reunion VSIX
+2.	Add package reference to Reunion
+At this point, the developer can stop and have an unpackaged app, with the ability to use MRT. None of the MSIX package tooling will run, because there is no existence of a Package.appxmanifest file. If the developer would like to create a packaged Desktop app, then they do the following:
+3.	Add Package.appxmanifest file to the project
+After this step, if they’d prefer to create a Sparse Package, they would add this to their project file:
+<WindowsPackageType>Sparse</WindowsPackageType>
+Question: could we update try convert to give existing Win32 apps Sparse by default?
+
+### 5.1.2 - Migrating Existing UWP Apps
```

Comment:
5.1.1 = Migrating Existing Win32 Apps
5.1.2 = Migrating Existing UWP Apps

What about existing Packaged Desktop apps? VS projects creating DesktopBridge apps are neither Win32 nor UWP. Do you intend to support migrating those?